internal_event_handlers.go

```diff
@@ -89,6 +88,7 @@ type (
 		enableLoggingInReplay bool // flag to indicate if workflow should enable logging in replay mode
 
 		metricsScope tally.Scope
+		hostEnv      *hostEnvImpl
```

Comment:
the test framework should work with the global host env as we are deprecate the private host env.